src/breakfast_b/morning_b/addition/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="../bundle.css">
```

Comment:
Because of the way the files are built, to use the css file, this line should be

`<link rel="stylesheet" href="../../../bundle.css">`